civihr_default_theme/scss/components/_chr-grid.scss

```diff
@@ -0,0 +1,104 @@
+/*
+util classes
+*/
+.chr-separator {
+    background-color: rgb(237, 237, 237);
+    height: 2px;
+}
+
+.chr-table__borderless tbody tr td,
+.chr-table__borderless tbody tr th,
+.chr-table__borderless thead tr th,
+.chr-table__borderless thead tr td,
+.chr-table__borderless tfoot tr th,
+.chr-table__borderless tfoot tr td {
+    border: none;
+}
+
+.chr-container {
+    margin-top: 60px;
+    /*background-color: rgb(247, 247, 249);*/
+}
+/*
+.chr-container__title {
+
+}
+*/
+.chr-container__row {
+  background-color: #f7f7f9;
+  padding: 24px;
+}
+.chr-container__row--summary {
+  height: 70px;
+  padding-top: 6px;
+  padding-bottom: 20px;
+}
+.chr-container__row__cell {
+  margin: 24px;
+}
+
+.chr-container__row__cell--separate {
+  box-shadow: 0px 6px 20px 0px rgba(0, 0, 0, 0.03),0px 8px 17px 0px rgba(0, 0, 0, 0.03);
```

Comment:
There is a `$crm_box-shadow-global` variable defined in *_variables.scss*. Although the value is slightly different, would it still work? Can we try to have the elements on the page always using the same shadow?